code/game/objects/structures/crates_lockers/closets/wardrobe.dm

```diff
@@ -412,7 +412,7 @@
 	new /obj/item/clothing/under/tactical(src)
 	new /obj/item/clothing/suit/armor/tactical(src)
 	new /obj/item/clothing/head/helmet/tactical(src)
-	new /obj/item/clothing/mask/balaclava/tactical(src)
+	new /obj/item/clothing/mask/balaclava/white(src)
```

Comment:
Isn't the white one going to look weird with the other stuff from that closet. (Wouldn't the green/gray/black fit better?)